ts/annotations/controllable/markerMixin.ts

```diff
@@ -0,0 +1,227 @@
+/* *
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+import H from './../../parts/Globals.js';
+
+/**
+ * Internal types.
+ * @private
+ */
+declare global {
+    namespace Highcharts {
+        interface AnnotationMarkerMixin {
+            markerEndSetter(this: SVGElement, value: string): void;
+            markerStartSetter(this: SVGElement, value: string): void;
+            setItemMarkers(this: AnnotationControllablePath, item: AnnotationControllablePath): void;
+        }
+        interface AnnotationChart {
+            afterGetContainer(): void;
+        }
+        interface Options {
+            defs?: Dictionary<SVGDefinitionObject>;
```

Comment:
Aren't `defs` already defined? See: https://api.highcharts.com/highcharts/defs

Reply:
Yes, but this is about the internal namespace, not about the API documentation.